CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `Recommender`s now share their core logic via their base class
 
+### Fixed
```

Comment:
@AVHopp @RimRihana Id appreciate if you could quickly confirm whether this fixes the situation you encountered

Reply:
Now the Tests are running when executing 'tox -e fulltest-py311' (before they did not) 

- but the compiler is still complaining for not finding the pwd 

![image](https://github.com/emdgroup/baybe/assets/118910837/5916458d-cc5c-48f6-864a-82b95590c00f)

- +the streamlint test fails 
![image](https://github.com/emdgroup/baybe/assets/118910837/a35e46e5-d8d6-408e-9473-50cadf4f7705)

Reply:
can you tell me how you installed baybe? the exception you show should be caught with the new code so that should not really be possible

Reply:
also, Im confused :). You said the test run but then wheres the screenshot with the faling test from?

Reply:
I already had baybe installed just checked out to your branch and run the tests.
Do you mean this Screenshot? 
<img width="866" alt="Unbenannt1" src="https://github.com/emdgroup/baybe/assets/118910837/b12364ce-742a-4790-93ac-94a5d9a060d5">

Reply:
I think what Rim means is "tox starts the tests". Before the fix, it did already fail before even executing any tests if I remember correctly.

Reply:
sorry I misread, the tests didnt even execute before -> but now they do -> good

1 of the 3 fails is related to hypothesis -> unrelated

2 of the 3 fails are streamlit tests -> i need to investigate but suspect it might be still unrelated

But acutally can you just import baybe and execute some random code?  OR run one of the examples? Dont forget that you need to delete the environment variable that you might have et to fix the original problem -> now all of it should work without you doing that fix, and we need to test that

Reply:
Yes exactly, before no tests were executed 
![image](https://github.com/emdgroup/baybe/assets/118910837/51e8bd6f-961f-4ed8-8d73-12d91fd11d1b)

I tried to reinstall baybe and rerun the tests: 
the bug did not show up, so I think the problem is solved :)
![image](https://github.com/emdgroup/baybe/assets/118910837/3077a959-6978-489a-857a-6cbe03d7f66c)

Reply:
> please execute `python -c "from baybe.telemetry import is_enabled, get_user_details; print(is_enabled(), get_user_details())"` from and environment where you did not adjust the username variables etc

Sorry the comments were not synchronized and did not see your comment.
This is what i get after executing the command: 
![image](https://github.com/emdgroup/baybe/assets/118910837/a96e0c0a-ba30-4447-88f5-af293621efdf)

Reply:
thanks, so we still need to confirm that you do not have any of the relevant environment variables set, because this was the original error. i suspect you did not reset them, which is why the `user` in your output is something else than `UNKNOWN`

please can you share the output of `SET` in your windows console? I cant really test it myself but we need to display all enviroment vaiables in your Windows console where you execute Python, it might be a different command

Reply:
If the output is the one from your last screenshot, you should not have encountered any error in the first place. The function that causes the problem on windows only tries to import the unavailable `cwd` when it cant find certain user variables:
![image](https://github.com/emdgroup/baybe/assets/17951239/05d7ca5b-fd06-487a-a8bd-2a05351cc95b)

But you have `USERNAME` sepcified, so that error should not have occurred. 

I guess you never encountered the issue in using baybe in an example or in a code execution, but only in tests? Can you confirm that?

This would make sense as probably these tests use environments that dont have these env variables, hence the error on windows. If you confirm together with the now-functioning tests we can conclude this PR

Reply:
Yes you are right! I never had problems while running a normal example 👍 
The problem only appeared when I wanted to run the fulltests, probably because baybe creates a new environment only for the tests.